pkg/scheduler/nodeinfo/node_info.go

```diff
@@ -388,6 +388,9 @@ func (n *NodeInfo) AllocatableResource() Resource {
 	if n == nil {
 		return emptyResource
 	}
+	if n.allocatableResource == nil {
```

Comment:
I am not sure this is the cause of the panic. If it was, shouldn't we see the panic when de-referencing at line 391?

According to #86908, the panic is at ```pkg/scheduler/framework/plugins/noderesources/resource_limits.go:120```